app/jobs/sync_synthetic_data_job.rb

```diff
@@ -0,0 +1,23 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class SyncSyntheticDataJob < BaseJob
+  include NotifierConfig
+  attr_accessor :send_notifications
+
+  def initialize
+    setup_notifier('Synthetic Data')
+    super
+  end
+
+  def perform
+    @notifier.ping('Processing synthetic data') if @send_notifications
+
+    GrdaWarehouse::Synthetic::Event.hud_sync
```

Comment:
Is the intention here we'd add a new call for each class?  (Assessment being the next one?)

Reply:
Yes.